demisto_sdk/commands/content_graph/strict_objects/playbook.py

```diff
@@ -0,0 +1,293 @@
+from typing import Any, Dict, List, Optional
+
+from pydantic import Field, constr
+
+from demisto_sdk.commands.common.constants import MarketplaceVersions
+from demisto_sdk.commands.content_graph.strict_objects.common import (
+    DESCRIPTION_DYNAMIC_MODEL,
+    FORM_DYNAMIC_MODEL,
+    ID_DYNAMIC_MODEL,
+    KEY_DYNAMIC_MODEL,
+    LEFT_DYNAMIC_MODEL,
+    MESSAGE_DYNAMIC_MODEL,
+    NAME_DYNAMIC_MODEL,
+    PLAYBOOK_INPUT_QUERY_DYNAMIC_MODEL,
+    REQUIRED_DYNAMIC_MODEL,
+    RIGHT_DYNAMIC_MODEL,
+    SCRIPT_ARGUMENTS_LOWER_CASE_DYNAMIC_MODEL,
+    SCRIPT_ARGUMENTS_UPPER_CASE_DYNAMIC_MODEL,
+    SCRIPT_ID_DYNAMIC_MODEL,
+    VALUE_DYNAMIC_MODEL,
+    BaseStrictModel,
+    create_model,
+)
+
+TASKS_REGEX = r"^[0-9]+(:?(xsoar|marketplacev2|xpanse|xsoar_saas|xsoar_on_prem))?$"
```

Comment:
Let's replace this list with `"|".join(MarketplaceVersions)`, in case we add another one day